DateTime.js

```diff
@@ -99,14 +99,14 @@ var Datetime = React.createClass({
 			inputValue = '';
 		else
 			inputValue = date || '';
-
+        
 		return {
 			updateOn: updateOn,
 			inputFormat: formats.datetime,
 			viewDate: viewDate,
 			selectedDate: selectedDate,
 			inputValue: inputValue,
-			open: props.open
+			open: props.open,
```

Comment:
Hi,  it was a left over from previous commit which added code there.  Linter won't catch this as its a common practice.  I'll remove it when I remove the extra spaces that were also include by the editor I am using.